Dockerfile

```diff
@@ -38,8 +38,6 @@ RUN cd /opt/cs3 && \
 
 RUN fix-permissions "/home/${NB_USER}"
 
-ENV JUPYTER_ENABLE_LAB = 1
```

Comment:
Why to remove this?

I guess that without this, the container would start with the classic UI, not jupyterlab. If we implement a plugin for jupyterlab, then it makes sense to run jupyterlab by default, and thus to keep this variable.

Reply:
this var was commented before and I deleted it by mistake

Reply:
I'm not sure what do you mean by "before".

I see it is enabled in branch master. I think it's good to keep it enabled.